src/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -238,7 +237,6 @@ class StockMovementItem {
         }
 
         StockMovementItem stockMovementItem = new StockMovementItem()
-        stockMovementItem.id = requisitionItemId
```

Comment:
This would create duplicate line items if the same file is imported multiple times.

Reply:
@jmiranda I've added changed commit, could you review?